setup.py

```diff
@@ -52,6 +52,7 @@
         'simplejson',
         'spectra>=0.0.10',
         'networkx' + ('<2.3' if sys.version_info[0:2] < (3, 5) else ''),  # pin for py<3.5
+        'beautifulsoup4',
```

Comment:
As this is used by travis only, and not needed for MultiQC execution, you don't need this here. You can just do a `pip install beautifulsoup4` in the travis setup instead.